src/components/sync-summary/SyncSummary.jsx

```diff
@@ -185,14 +186,7 @@ class SyncSummary extends React.Component {
                             </ExpansionPanelSummary>
 
                             <ExpansionPanelDetails>
-                                <ReactJson
-                                    src={{
-                                        results: response.results,
-                                        syncReport: response.syncReport,
-                                    }}
-                                    collapsed={2}
-                                    enableClipboard={false}
-                                />
+                                <ReactJson src={response} collapsed={2} enableClipboard={false} />
```

Comment:
The problem here is that we print injected internal class properties that should be hidden (as an example selected property from ObjectsTable). I thought of adding a lodash pick of results and syncReport. What do you think?

Reply:
I see! Then, yes, we need to recover a `_.pick` here.

Reply:
Will update the PR in a few minutes so it's ready to merge